Rust.cmake

```diff
@@ -0,0 +1,39 @@
+# Dependencies that are downloaded using the CPM package manager
+include("${CMAKE_CURRENT_LIST_DIR}/CPM.cmake")
```

Comment:
Will this be a problem if CPM is already included? Do we need something like https://cmake.org/cmake/help/latest/command/include_guard.html ?